### ISSUE TITLE
Add option to disable sorting CPU usage chart

### ISSUE THREAD

**Issue by SoMuchForSubtlety** [fflewddur/tophat#154]
The CPU usage chart started being sorted by usage recently. I would prefer the previous behaviour (sort by logical CPU number).

**Comment by fflewddur** [comment]
I changed this because I found it hard to meaningfully see differences in CPU use when needing to visually scan all of the bars. For example, it's common for long-running processes to get moved from core to core, so different bars would suddenly spike in TopHat, but it's just the OS shifting the workload to a different core. The redesign is intended to fix that. Is there a reason you find it more useful to have these sorted by logical core?

**Comment by SoMuchForSubtlety** [comment]
Yes, I found it useful to see when a process switched to a different core. I guess a toggle in the settings would allow for both behaviours.
